setup.cfg

```diff
@@ -1,2 +1,2 @@
 [pep8]
-exclude = ./venv,./bower_components,./digital_marketplace_frontend_toolkit
```

Comment:
I've used the same file we have in other frontend apps. I couldn't find `digital_marketplace_frontend_toolkit` after running frontend build tasks, but as @quis mentioned it seems to be in `./bower_components/digital_marketplace_frontend_toolkit` now, which is ignored by `./bower_components`